packages/env/lib/env.js

```diff
@@ -91,21 +106,9 @@ module.exports = {
 				}
 			} )(),
 
-			...config.pluginSources.map( ( source ) =>
-				downloadSource( source, {
-					onProgress: getProgressSetter( source.basename ),
-					spinner,
-					debug: config.debug,
-				} )
-			),
-
-			...config.themeSources.map( ( source ) =>
-				downloadSource( source, {
-					onProgress: getProgressSetter( source.basename ),
-					spinner,
-					debug: config.debug,
-				} )
-			),
+			...getSourceDownloads( config.pluginSources, downloader ),
+			...getSourceDownloads( config.muPluginsSources, downloader ),
+			...getSourceDownloads( config.themeSources, downloader ),
```

Comment:
Not sure if it's because I'm dumb (I am!) or because it's Monday morning (it is!) but I struggled to parse this at first. Similar to my other comment above, I'd be tempted to use `castArray` here to remove `getSourceDownloads`.

```js
...castArray( config.pluginSources ).map( downloader ),
...castArray( config.muPluginsSources ).map( downloader ),
...castArray( config.themeSources ).map( downloader ),
```

Reply:
cast array would work very nicely here! I didn't know about it before :)

Reply:
and done